src/components/select/CdrSelect.vue

```diff
@@ -58,11 +57,11 @@ const props = defineProps({
    */
   background: backgroundProps,
   /**
-   * Sets the component's size; values can target responsive breakpoints. Example `small@lg`
-   * @demoSelectMultiple false
-   * @values small, medium, large
+   * Sets the component's size; values can target responsive breakpoints. Example `large@lg`
+   * @demoSelectMultiple true
+   * @values large
   */
-  size: sizeProps,
+  size: String,
```

Comment:
An example of where a component was using the common `size` prop util but should't have been because it was introducing invalid options.